scapy/contrib/ikev2.py

```diff
@@ -743,6 +743,20 @@ class IKEv2_payload_Encrypted(IKEv2_class):
     ]
 
 
+class IKEv2_payload_CP(IKEv2_class):  # RFC 7296, section 3.15
+    name = "IKEv2 Configuration"
+    overload_fields = {IKEv2: {"next_payload": 46}}
+    fields_desc = [
+        ByteEnumField("next_payload", None, IKEv2_payload_type),
+        ByteField("res", 0),
+        FieldLenField("length", None, "load", "H", adjust=lambda pkt, x:x + 8),
+        ByteEnumField("CFGtype", 1, {1: "CFG_REQUEST", 2: "CFG_REPLY", 3: "CFG_SET", 4: "CFG_ACK", 11: "Key"}),  # noqa: E501
+        ByteEnumField("res2", 0, {0: "Unused"}),
+        ShortEnumField("res3", 0, {0: "Unused"}),
+        StrLenField("load", "", length_from=lambda x: x.length - 8),
```

Comment:
The load is actually a list of TLVs, the so called Configuration Attributes ([RFC 7296, section 3.15.1](https://www.rfc-editor.org/rfc/rfc7296.html#section-3.15.1)). Is there a generic or recommended way how dissect those TLVs in Scapy (e.g, some sort of  `FieldListField` list of `TLVField`s, each `TLVField` containing a nested `type`, `length`, and `value` field?)

Reply:
Typically we just use a `PacketFieldList` with a custom packet. You could make an `IKEv2_CP_TLV` class or something.

Reply:
Thanks for the hint, according to the documentation it seems to be an easy straightforward task. I'll go back to draft mode until it's done, but feel free to review the rest in the meanwhile, this is the only missing feature I intend to add.

Reply:
done